src/components/ContentSidebar/MetadataSidebar.scss

```diff
@@ -1,9 +1,30 @@
+@import '../variables';
+
 .be.bcs.bcs-metadata {
+    .bcs-metadata-header {
+        align-items: center;
+        border-bottom: 1px solid $off-white;
+        display: flex;
+        height: 60px;
```

Comment:
It looks like there's duplication here and in `SidebarTitle.scss`. Should there be two components, a header (display, height, margin, etc.) and a title (font-size and font-weight)? That way the size of the header will always be consistent between panels.